Src/CSharpier.Tests/FormattingTests/TestFiles/Comments_EdgeCase.expected.cst

```diff
@@ -6,21 +6,42 @@ public class ClassName
 
     public void Method()
     {
-        /* multiline
-         *  multiline
-         */
+        /* this doesn't work well because the indentation is changing
+     *  multiline
+     */
         var variable = true;
     }
 
     /// <summary> no tabs here
     /// Tabs here
     /// </summary>
     public class ClassName { }
+
+    /* this works fine
+     * because indentation does not change
+     */
+    public class ClassName2 { }
+
+    /*
+    public string String;
+    */
 }
 
 public interface Interface
 {
     /// <summary>
     ///	</summary> tab here before <
     void Method();
+
+    /* this also doesn't work well because of turning 2 tabs into 4 spaces
+		and it keeps the tabs here, which is maybe what we want
```

Comment:
I think most editors will automatically turn tabs into spaces. So we'll probably receive spaces :shrug:

Reply:
yeah this feels super edge case, there was one file in moq4 that had this issue.